### ISSUE TITLE
Removing redundant imports in the practice app

### ISSUE THREAD

**Issue by egetaga** [bounswe/bounswe2022group1#187]
In the practice app code, there are a bunch of redundant imports. Some of them, like "from asyncio.windows_events import NULL" are decreasing the cross-platform compatibility of the code, since "asyncio.windows_events" are not present in the operating systems other than windows, the app cannot run on OS X, or linux. 

Moreover, "NoneType" does not require imports now in current versions of python (for example python3.9.9). So, "from types import NoneType" cannot resolve and raised an error in python3.9.9 in my local. 

In summary, the tasks are as follows:

- Remove "from asyncio.windows_events import NULL" statements in all the **.py** files.
- Remove "from types import NoneType" statements in all the **.py** files.
